.travis/install.sh

```diff
@@ -42,3 +42,5 @@ py36)
     cython "camb>=1"
   ;;
 esac;
+
+pip install https://github.com/JoeMcEwen/FAST-PT/archive/v3.0.2.tar.gz --user
```

Comment:
Don't use a user install here.

Reply:
You should add --no-deps as well.

Reply:
Thanks!

Reply:
@beckermr It still doesn't seem to be able to find fastpt. pip installs successfully, but it still can't find it. I imagine this has to do with miniconda, but I'm not familiar with that at all. Let me know if you have any thoughts.

Reply:
OK. Let me dig into it.

Reply:
Try this.

Reply:
I put in a stupid bug. Should work now.